examples/worker-interaction/worker_interaction.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how workers interact with each other by sending and receiving messages.
```

Comment:
Is there a way we can add the asynchronous keyword to the description?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/544/commits/42146f5468ef8eb6677cc70e3936e03cfedfd2f6